rdkcompositor.cpp

```diff
@@ -150,33 +151,67 @@ namespace RdkShell
         }
     }
 
-    bool RdkCompositor::createDisplay(const std::string& displayName, uint32_t width, uint32_t height)
+    bool RdkCompositor::loadExtensions(WstCompositor *compositor, const std::string& clientName)
```

Comment:
Let's have `loadExtensions()` return true on success and false on failure to follow the consistency of the other RDKShell apis.

Reply:
done